pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/catalog/PmdCatalogCategory.java

```diff
@@ -9,14 +9,20 @@
 	 * The path to the definition file for this category.
 	 */
 	private String path;
+	/**
+	 * The JAR containing this category's definition file.
+	 */
+	private String sourceJar;
```

Comment:
This is not required, right?  They don't have to have specified a jar file originally?

Reply:
Currently, PMD rules have to be defined in a JAR. We're not at the point where we could just accept the root directory of a Java project.